### PR TITLE
[feature-wip](MTMV) Support joining tables with views

### PR DESCRIPTION
**Pull request by adonis0147**
# Proposed changes

~~Issue Number: close #xxx~~

## Problem summary

Currently, some errors occur when we create a materialized view with multiple tables which include a inline view. We should support these scenarios.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [x] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by adonis0147** [comment]
run buildall

File: fe/fe-core/src/test/java/org/apache/doris/catalog/MultiTableMaterializedViewTest.java [hunk 1]
```diff
@@ -368,4 +368,146 @@ public void testCreateWithHint() throws Exception {
                         || result.contains("SELECT /*+ SET_VAR(query_timeout=3600, exec_mem_limit=1048576) */")
         );
     }
+
+    @Test
+    public void testCreateWithViews() throws Exception {
+        createTable("CREATE TABLE lineorder ("
```
Comment (SaintBacchus) [inline fe/fe-core/src/test/java/org/apache/doris/catalog/MultiTableMaterializedViewTest.java:374]: all the ssb mv stmt can be covered in this case?

File: fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java [hunk 2]
```diff
@@ -130,12 +130,8 @@ public void processCreateMaterializedView(CreateMaterializedViewStmt stmt)
 
     public void processCreateMultiTableMaterializedView(CreateMultiTableMaterializedViewStmt stmt)
             throws UserException {
-        // check db
-        Database db = stmt.getDatabase();
         // check cluster capacity
         Env.getCurrentSystemInfo().checkClusterCapacity(stmt.getClusterName());
```
[the remark this answers is not in the document]
Comment (adonis0147) [reply fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java]: Refined.

**Comment by adonis0147** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.12 seconds
 stream load tsv:          452 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         25 seconds loaded 2358488459 Bytes, about 89 MB/s
 stream load orc:          73 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230322142905_clickbench_pr_118935.html

**Review by morningman — APPROVED** [review]
LGTM

**Review by chenlinzhong — APPROVED** [review]
LGTM

**Pull request merged**